polars/polars-io/src/csv/buffer.rs

```diff
@@ -161,11 +168,12 @@ impl ParsedBuffer for Utf8Field {
         bytes: &[u8],
         ignore_errors: bool,
         needs_escaping: bool,
+        missing_is_null: bool,
     ) -> PolarsResult<()> {
         if bytes.is_empty() {
             // append null
             self.offsets.push(self.data.len() as i64);
-            self.validity.push(false);
+            self.validity.push(!missing_is_null);
```

Comment:
That's a smart one. Makes it branchless indeed. :)

Reply:
At the moment my approach to Rust is _"run the code in my head as best I can, and stare at it until it bends to my will"_ 🤣 (maybe I'll ask on the Discord if anyone else has set up a Clion build env; would be nice to be able to step through and get a better sense of it).